package.json

```diff
@@ -41,22 +41,23 @@
     "pretest-e2e-ios": "npm run build",
     "test-e2e-ios": "node ./scripts/test.e2e.ios.js",
     "test-all": "node ./scripts/test.all.js",
-    "release": "node ./scripts/release.js"
+    "release": "node ./scripts/release.js",
+    "prepublish": "npm run build"
   },
   "peerDependencies": {
     "react": "*",
     "react-native": "*"
   },
   "dependencies": {
-    "lodash": "4.x.x",
-    "prop-types": "15.x.x",
-    "tslib": "1.x.x",
-    "@types/react": "16.x.x",
-    "@types/react-native": "0.49.x",
+    "@types/jest": "22.x.x",
```

Comment:
Do we need to reorder these? It seems likely to cause endless headaches as we merge wix v2.

Reply:
We don't, but I didn't manually reorder these. They were auto-alphabetized after I `yarn`'d. I can revert.